src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/Helpers.cs

```diff
@@ -75,58 +75,122 @@ public static byte[] ValueAsAscii(this Oid oid)
+                if (!Interop.crypt32.CryptDecodeObjectPointer(
+                    CertEncodingType.All,
+                    lpszStructType,
+                    encoded,
+                    encoded.Length,
+                    CryptDecodeObjectFlags.None,
+                    null,
+                    ref cb))
+                {
                     throw Marshal.GetLastWin32Error().ToCryptographicException();
+                }
 
                 byte* decoded = stackalloc byte[cb];
-                new Span<byte>(decoded, cb).Clear();
-                if (!Interop.crypt32.CryptDecodeObjectPointer(CertEncodingType.All, lpszStructType, encoded, encoded.Length, CryptDecodeObjectFlags.None, (byte*)decoded, ref cb))
+
+                if (!Interop.crypt32.CryptDecodeObjectPointer(
+                    CertEncodingType.All,
+                    lpszStructType,
+                    encoded,
+                    encoded.Length,
+                    CryptDecodeObjectFlags.None,
+                    decoded,
+                    ref cb))
+                {
                     throw Marshal.GetLastWin32Error().ToCryptographicException();
+                }
 
                 receiver(decoded, cb);
             }
         }
 
-        public static bool DecodeObjectNoThrow(this byte[] encoded, CryptDecodeObjectStructType lpszStructType, DecodedObjectReceiver receiver)
+        public static bool DecodeObjectNoThrow(
+            this byte[] encoded,
+            CryptDecodeObjectStructType lpszStructType,
+            DecodedObjectReceiver receiver)
         {
             unsafe
             {
                 int cb = 0;
 
-                if (!Interop.crypt32.CryptDecodeObjectPointer(CertEncodingType.All, lpszStructType, encoded, encoded.Length, CryptDecodeObjectFlags.None, null, ref cb))
+                if (!Interop.crypt32.CryptDecodeObjectPointer(
+                    CertEncodingType.All,
+                    lpszStructType,
+                    encoded,
+                    encoded.Length,
+                    CryptDecodeObjectFlags.None,
+                    null,
+                    ref cb))
+                {
                     return false;
+                }
 
                 byte* decoded = stackalloc byte[cb];
-                new Span<byte>(decoded, cb).Clear();
-                if (!Interop.crypt32.CryptDecodeObjectPointer(CertEncodingType.All, lpszStructType, encoded, encoded.Length, CryptDecodeObjectFlags.None, (byte*)decoded, ref cb))
+
+                if (!Interop.crypt32.CryptDecodeObjectPointer(
+                    CertEncodingType.All,
+                    lpszStructType,
+                    encoded,
+                    encoded.Length,
+                    CryptDecodeObjectFlags.None,
+                    decoded,
+                    ref cb))
+                {
                     return false;
+                }
 
                 receiver(decoded, cb);
             }
+
```

Comment:
All of the above changes are purely formatting, right?  I scanned them looking for differences but didn't see any, other than whitespace and removal of unnecessary casts.

Reply:
(Plus the Span.Clear removal, of course.)

Reply:
Yeah, the lines were longer than both the default github review window and my code window aperture in VS; so I chopped them.